tests/cpptests/coord_tests/test_cpp_io_runtime_ctx.cpp

```diff
@@ -14,6 +14,9 @@
 #include "rmutil/rm_assert.h"
 #include "redismodule.h"
 #include "info/global_stats.h"
+extern "C" {
```

Comment:
I think u can just add the cplusplus extension inside concurrent_ctx.h?

Reply:
ohh gotcha
moved